client/wfprev-war/src/main/angular/src/app/components/edit-project/project-details/evaluation-criteria/evaluation-criteria.component.html

```diff
@@ -0,0 +1,75 @@
+<div>
+    <mat-expansion-panel #panel="matExpansionPanel">
+        <mat-expansion-panel-header class="expansion-panel-header">
+          <mat-panel-title class="project-title">
+            <div class="custom-indicator">
+                <wfprev-expansion-indicator [isExpanded]="panel.expanded"></wfprev-expansion-indicator>
+            </div>
+            <span>Evaluation Criteria</span>
+          </mat-panel-title>
+          <mat-panel-description class="edit-evaluation-criteria">
+            <div (click)="openEvaluationCriteriaPopUp(); $event.stopPropagation()">
+              <img alt="edit-icon" class="icon" src="/assets/edit-icon.svg" />
+              <span>Edit</span>
+            </div>
+          </mat-panel-description>
+        </mat-expansion-panel-header>
+        <!-- Fuel Management -->
+        <div *ngIf="project?.projectTypeCode?.projectTypeCode === 'FUEL_MGMT' && evaluationCriteriaSummaries.length">
+          <div class="criteria-section">
+            <div class="section-label">
+              <span>Coarse Filter Total:</span>
+              <span>
+                {{ evaluationCriteriaSummaries[0]?.localWuiRiskClassCode?.weightedRank
```

Comment:
With the constant use of `evaluationCriteriaSummaries[0]` might make sense to move that value into an explicit variable, rather than carrying the [0] around everywhere.

Reply:
Also, if we're only using the first object in the array, why is it an array at all?

Reply:
Updated, it should be just one object, changed to use evaluationCriteriaSummary